docs/installation.md

```diff
@@ -0,0 +1,243 @@
+# Installing ansible-navigator with execution environment support
+
+[podman_url]: https://podman.io/
+
+```{contents}
+:local:
+```
+
+## Linux
+
+[podman_install_url]: https://podman.io/getting-started/installation
```

Comment:
It's quite uncommon to use such identifiers in MD detached link definitions. Just use normal text as it is used in the prose.